Source/SPAppController.m

```diff
@@ -102,6 +102,16 @@ - (id)init
 		[fileManager createDirectoryAtPath:[NSHomeDirectory() stringByAppendingPathComponent:@"tmp"] withIntermediateDirectories:true attributes:nil error:nil];
 		[fileManager createDirectoryAtPath:[NSHomeDirectory() stringByAppendingPathComponent:@".keys"] withIntermediateDirectories:true attributes:nil error:nil];
 
+		// Ignore Dark Theme when needed
+ 		if (@available(macOS 10.14, *)) {
+			NSInteger style = [[NSUserDefaults standardUserDefaults] integerForKey:SPThemeStyle];
+			if (style == 1) {
+ 				NSApp.appearance = [NSAppearance appearanceNamed:NSAppearanceNameAqua];
+			} else if (style == 2) {
```

Comment:
What if the style changes to automatic, then the standardUserDefaults value is removed? :)

Reply:
Sorry, didn't get you. Are you mean about some sort of `free`'ing of `[NSUserDefaults standardUserDefaults]` construction? Honestly, I am still too stupid in ObjC to understand what happens with the memory here :(
If the question is about what `if` branch executes in "Automatic" case, then no one － we are set no overridden style (also `nil` is ok), and it changes as earlier.